### PR TITLE
Redirects deployer, httpd configuration

### PR DESCRIPTION
**Pull request by Sanne**

### CODE REVIEW

**Comment by gunnarmorling** [comment]
We should rename the folder from "redirects" to something like "server-config", reflecting the more general semantics it has now. Don't forget to adapt .awestruct_ignore.

File: redirects/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+scp staginginrelationto.conf ci.hibernate.org:~
+
+cat file-opening-production.conf > inrelationto.conf
+cat .htaccess_misc >> inrelationto.conf
+cat .htaccess_bloggers >> inrelationto.conf
+cat .htaccess_posts >> inrelationto.conf
+cat file-closing.conf >> inrelationto.conf
+scp inrelationto.conf ci.hibernate.org:~
+
+ssh -t ci.hibernate.org "sudo cp staginginrelationto.conf /etc/httpd/conf.d/staginginrelationto.conf && sudo cp inrelationto.conf /etc/httpd/conf.d/inrelationto.conf && sudo service httpd restart"
```
Comment (gunnarmorling) [inline redirects/deploy.sh]: Better use reload than restart?

File: redirects/file-opening-production.conf [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<VirtualHost *:80>
+  ServerName in.relation.to
+  ServerAdmin webmaster@localhost
```
Comment (gunnarmorling) [inline redirects/file-opening-production.conf]: Or our actual admin address?

File: redirects/deploy.sh (hunk 1 above)
Comment (Sanne) [inline redirects/deploy.sh]: thanks for that. when did they change it to 'reload' ?! I was looking for the 'graceful' but it was not accepted. I guess thanks systemd..

File: redirects/file-opening-production.conf (hunk 2 above)
Comment (Sanne) [inline redirects/file-opening-production.conf]: +1

**Comment by Sanne** [comment]
I've applied the suggestions and merged it.. thanks!

**Comment by hferentschik** [comment]
> We should rename the folder from "redirects" to something like "server-config", reflecting the more general semantics it has now

+1

**Pull request closed without merging**